packages/rich-text-editor/docs/RichTextEditor.stories.tsx

```diff
@@ -92,3 +92,32 @@ WithBadData.args = {
     { name: "link", group: "link" },
   ],
 }
+
+export const WithDescriptionAndValidationMessage = args => {
+  const [rteData, setRTEData] = useState<EditorContentArray>(dummyContent)
+  return (
+    <RichTextEditor
+      value={rteData}
+      onChange={data => setRTEData(data)}
+      {...args}
+    />
+  )
+}
+
+WithDescriptionAndValidationMessage.storyName =
```

Comment:
Suggestion only, but technically you could do this pattern:

```suggestion
// Although you'd probably want to rename `WithData` to a name with `Template` at the end
WithDescriptionAndValidationMessage.storyName = WithData.bind({})
```